docs/content/any/getting-started/quickstart/index.md

```diff
@@ -186,7 +186,7 @@ will help you out.
 {{% callout "What's next" "blue" %}}
 
 - Explore how to [add Oso to an application](application).
-- [Use Oso Roles](/guides/new-roles) to add Role-Based Access Control to your application
+- [Use Oso Roles](/guides/roles) to add Role-Based Access Control to your application
```

Comment:
follow-up: standardize capitalization of "Role-Based Access Control" vs. "role-based access control" across our docs